Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/statement/opaqueBlock.dfy

```diff
@@ -0,0 +1,128 @@
+// RUN: ! %verify %s &> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Foo() returns (x: int)
+  ensures x > 4 
+{
+  x := 1;
+  var y := 1;
+  opaque
+    ensures x > 3 
+  {
+    x := x + y;
```

Comment:
Have you considered testing a `AssignOrReturnStmt`?

Reply:
Good callout. Added some more tests and found bugs.

Reply:
Sorry for not thinking of it sooner: What about `AssignSuchThatStmt`?

Reply:
Added it and found another bug 🎉

Reply:
Sorry for the dripping of ideas: What about multiple LHSs and Patterns?

Reply:
What sort of patterns do you mean? Added the parallel assignment